.travis.yml

```diff
@@ -113,8 +123,13 @@ matrix:
     - { rvm: *ruby_2_4,  env: *rails_master }
 
   allow_failures:
+    # Rails 5.2.4.1 used safe-navigation operator (&.), which caused ruby 2.2 to fail
+    # https://buildkite.com/rails/rails/builds/65743#e3a74619-d8f6-493f-b590-1203028167d3
+    - { rvm: *ruby_2_2,  env: *rails_5_2 }
```

Comment:
instead of allowing failures we should exclude them

Reply:
it seems like rails team will fix it in coming days (as rails 5-2-stable branch still supports ruby 2.2). so we may keep supporting it